frontend/src/js/controllers/authCtrl.js

```diff
@@ -7,10 +7,12 @@
         .module('evalai')
         .controller('AuthCtrl', AuthCtrl);
 
-    AuthCtrl.$inject = ['utilities', '$state', '$rootScope', '$timeout'];
+    AuthCtrl.$inject = ['utilities', 'configService','$state', '$rootScope', 'stringToTemplate'];
```

Comment:
Inject these in order like this. 
`['utilities', 'configService', 'stringToTemplate', '$state', '$rootScope'];`